src/storage/migration.rs

```diff
@@ -8,40 +8,46 @@ use std::{
 };
 
 use anyhow::{anyhow, Context, Result};
-use giganto_client::ingest::log::SecuLog;
+use giganto_client::ingest::{
```

Comment:
It would be preferable to modify the code to access it using crate::storage instead of referencing the structure directly from giganto-client in order to clearly indicate the purpose of changing the DB schema. (ex)`crate::storage::Netflow5`)

Reply:
Done